include/Clusterer.h

```diff
@@ -2,6 +2,8 @@
 #define CLUSTERER_H
 
 #include "CTSEvent.h"
+#include "Cluster.h"
+#include <vector>
```

Comment:
Separate the framework includes from std includes with an empty line. Also ROOT includes should be separated in this way so that one has up to 3 blocks of includes. This is nice for readability. I just saw that you did it in the calibrateCTSEvents macro.